internal/testutils/mock/mock_openshift_client.go

```diff
@@ -4,46 +4,61 @@ import (
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 )
 
+// OpenShiftClient is a client for OpenShift API
+// It implements client.OpenShiftClient
 type OpenShiftClient struct {
 	IdleState       int
 	IdleCallCount   int
 	UnIdleCallCount int
 }
 
+//Idle scales down the jenkins pod in the given OpenShift namespace
 func (c *OpenShiftClient) Idle(namespace string, service string) error {
 	c.IdleCallCount++
 	return nil
 }
 
+// UnIdle scales up the jenkins pod in the given OpenShift namespace
 func (c *OpenShiftClient) UnIdle(namespace string, service string) error {
 	c.UnIdleCallCount++
 	return nil
 }
+
+//IsIdle returns `JenkinsIdled` if a service in OpenShit namespace is idled,
```

Comment:
No, this is a mock impl. You cannot just take the godoc from the interface. This method can return whatever you want it to return.